packages/react/examples/codesandbox/components/ButtonGroup/package.json

```diff
@@ -4,20 +4,20 @@
   "version": "1.0.0",
   "main": "index.html",
   "scripts": {
-    "start": "parcel index.html --open",
+    "start": "parcel serve -p 9000 index.html --no-hmr",
     "build": "parcel build index.html"
   },
   "dependencies": {
     "@carbon/ibmdotcom-react": "latest",
     "@carbon/ibmdotcom-styles": "latest",
-    "@carbon/icons-react": "10.11.0",
-    "react": "16.9.0",
-    "react-dom": "16.9.0"
+    "@carbon/icons-react": "10.13.0",
+    "react": "16.13.0",
+    "react-dom": "16.13.0"
```

Comment:
@asudoh do you see any issues with us using a different version in our examples from our codebase?

Reply:
Good question @kennylam, I smoke-tested all examples and fixed issues I found (none of them seems to have been related to dependency update). The only remainder is #2999, where we need to wait for a new release.

Reply:
Great thanks!